about.html

```diff
@@ -13,56 +13,85 @@
 
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
+     <header class="top-nav-bar">
+            <div class="top-flex-item"><img src=".\img\lambda-black.png"></div>
+            <div class="top-flex-item"><a href="#">Home</a></div>
+            <div class="top-flex-item"><a href=".\about.html">About</a></div>        
+            <div class="top-flex-item"><a href="#">Products</a></div>
+            <div class="top-flex-item"><a href="#">Blog</a></div>
+            <div class="top-flex-item"><a href="#">Content</a></div>
+        </header>
+        <div class="jumbo-photo">
+            <div class="top-flex-item"><img src=".\img\jumbo-about.png"></div>
```

Comment:
When using the <img /> it is best practice to include an "alt" attribute. alt allows you to add a description of the image in case you image doesn't load and it helps people using screen readers know what the image is